requirements/test.txt

```diff
@@ -11,7 +11,7 @@ threadpoolctl
 # scikit-umfpack  # circular dependency issues
 pooch
 hypothesis>=6.30
-array-api-strict>=2.0,<2.1.1
+array-api-strict>=2.3
```

Comment:
This got out of sync with `pyprojects.toml`. Ran `python tools/generate_requirements.py`.

Reply:
Maybe even `>=2.3.1`